core/src/main/scala/org/apache/spark/ui/env/EnvironmentPage.scala

```diff
@@ -19,9 +19,11 @@ package org.apache.spark.ui.env
 
 import javax.servlet.http.HttpServletRequest
 
+import scala.collection.mutable
```

Comment:
Nit: import `scala.collection.mutable.StringBuilder` and use `val execStr = new StringBuilder()` below